src/iso22133state.cpp

```diff
@@ -64,6 +64,7 @@ void ISO22133::State::handleEvent(TestObject& obj, const ISO22133::Events::Event
 		obj.state = obj.createUnknown();
 		break;
 	}
+	lock.unlock();
```

Comment:
unsure why this was required here but not in the other locations scoped_lock was replaced